README.md

```diff
@@ -92,6 +101,34 @@ Unless otherwise stated, the APIs with the same names as v1 APIs have a similar
 * Rate Limit Discovery Service (RLDS). This is the same as RLS in v1.
 * [Route Discovery Service (RDS)](api/rds.proto).
 
+In addition to the above APIs, an aggregation API will be provided to allow for
+fine grained control over the sequencing of API updates across discovery
+services:
+
+* [Aggregated Discovery Service (ADS)](api/ads.proto). While fundamentally Envoy
+  employs an eventual consistency model, ADS provides an opportunity to sequence
+  API update pushes and ensure affinity of a single management server for an
+  Envoy node for API updates. ADS allows one or more APIs to be delivered on a
+  single gRPC bidi stream by the management server, and within an API to have all
+  resources aggregated onto a single stream. Without this, some APIs such as RDS
+  and EDS may require the management of multiple streams and connections to
+  distinct management servers.
+
+  ADS will allow for hitless updates of configuration by appropriate sequencing.
+  For example, suppose *foo.com* was mappped to cluster *X*. We wish to change
+  the mapping in the route table to point *foo.com* at cluster *Y*. In order to
+  do this, a CDS/EDS update must first be delivered containing both clusters *X*
+  and *Y*.
+
+  Without ADS, the CDS/EDS/RDS streams may point at distinct management servers,
+  or when on the same management server at distinct gRPC streams/connections
+  that require coordination. The EDS resource requests may be split across two
+  distinct streams, one for *X* and one for *Y*. ADS allows these to be
+  coalesced to a single stream to a single management server, avoiding the need
+  for distributed synchronization to correctly sequence the update. With ADS,
+  the management server would deliver the CDS, EDS and then RDS updates on a
```

Comment:
Yeah good point about version numbers. OK let's just keep it simple. We can always add another API later if needed.